### ISSUE TITLE
Support for World Edit (.schem) files for Structures

### ISSUE THREAD

**Issue by chrismwiggs** [VolmitSoftware/Iris#71]
**Is your feature request related to a problem? Please describe.**
While the built-in Iris features for creating structures are good, as a server admin with builder used to world edit, it would be nice to be able to drop in a World Edit Schematic file and have Iris accept it/automatically port it over.

**Describe the solution you'd like**
Schematic files are nice because they're part of many people's existing workflow but they're also easy to edit and revise as part of this workflow. 

**Describe alternatives you've considered**
I've used other plugins, like CustomStructures to simplify the custom structure process; we had a lot of structures built already and I was able to just drop these .schem files into a folder, add some basic spawn criteria, and was done. Making changes was as easy as replacing the .schem file, which was nice + felt very natural for my usual admin workflow. 

**Additional context**
This is very much a nice-to-have, but I think would help in lowering the barrier to more customization!

**Comment by cyberpwnn** [comment]
Does the converter work? moving them into that convert folder in iris then using /iris convert? It works for EWG Object files, i think i added support for world edit but im not sure how well it works.

**Comment by svdgoor** [comment]
Convert was removed, @cyberpwnn could this be reintroduced?

**Comment by svdgoor** [comment]
Perhaps this can be added to the converter introduced with #444

**Comment by svdgoor** [comment]
Would be nice to have a converter again in general tho, with the new obj v3, #508

**Comment by cyberpwnn** [comment]
Yeah it will be. We cant directly support schematic files, they have to be converted. Schematic format is not suitable for generation as they store the entire cuboid of data, its very wasteful with trees for example.

**Comment by Skizzles** [comment]
I would love this feature of conversion, would make some stuff easier. Definitely adding the file and running a convert command would do.

**Comment by NextdoorPsycho** [comment]
It would be easier but a lot of work
